recipes/llama-cpp/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "b4570":
```

Comment:
We are missing this version in the `config.yml`, It appear in the linter as warning, but it advises not to handle that in this PR. I think that's an error in this case and we should change the linter advise.

Reply:
Good catch, my bad!